requirements.txt

```diff
@@ -1,6 +1,6 @@
 beautifulsoup4==4.9.1
 defusedxml==0.7.1
-pdfminer.six==20211012
+git+https://github.com/phantomcyber/pdfminer.six.git@20211012-fips
```

Comment:
This is the key change. 
Here, I replaced all MD5 usage with SHA256.

Reply:
Reference: https://github.com/splunk-soar-connectors/ssh/pull/1/files#diff-4d7c51b1efe9043e44439a949dfd92e5827321b34082903477fd04876edb7552R2